src/adaptors/brinc-finance/index.js

```diff
@@ -27,19 +27,20 @@ const brcContract = new web3.eth.Contract(BrcABI, BRC);
 const BLOCKS_PER_MONTH = 199384;
 const TOTAL_POOLS_WEIGHT = 568;
 const PoolNames = [
-  'BRC 7Days',
-  'BRC 30Days',
-  'BRC 90Days',
-  'BRC-gBRC 7Days',
-  'BRC-gBRC 30Days',
-  'BRC-gBRC 90Days',
+  'BRC (7Days)',
```

Comment:
please revert the change re (). 

on the frontend it will look as intended:
<img width="552" alt="Screenshot 2022-08-31 at 23 28 56" src="https://user-images.githubusercontent.com/74533329/187846761-4371e8c9-ef26-4538-8d0d-e28ba2736fd8.png">

Reply:
It's not really clear what your request is.

Do you want us to remove the ()? You asked to have them added in #48

Reply:
hey, sry: we made a few changes in the meantime, and we moved that info to the `poolMeta` field. i confirm: please remove () and keep it like it master

Reply:
They're removed now